src/main/resources/model/instance.json

```diff
@@ -54,7 +54,8 @@
       "type": "object",
       "properties": {
         "name": {
-          "index": "keyword",
+          "inventorySearchTypes": "contributors.name",
```

Comment:
existing contract for query - ```(contributors =/@name "harry") sortby title```, according to this change - UI should change query to ```(contributors.name ="harry1") sortby title````, but we agreed that we support existing contract 1 to 1

Reply:
Now we support both notations